lib/src/bindings/pthread.dart

```diff
@@ -10,29 +10,61 @@ final class pthread_mutex_t extends Opaque {}
 
 final class pthread_cond_t extends Opaque {}
 
+// final class pthread_timespec_t extends Opaque {}
+
+final class pthread_timespec_t extends Struct {
+  @Int64()
+  external int tv_sec;
+  
+  @Int64()
+  external int tv_nsec;
+}
+
+/// mutex_init
```

Comment:
These comments are redundant. They don't add anything.

Reply:
they were added to help visual parsing of the code not to add documentation.

Reply:
The at the very least make this consistent: 

- in `pthread.dart` you added a comment at every function, in `winapi` you did not. 
- in `pthread.dart` you use function name (minus `pthread_`), in `winapi` you use human readable categories instead.

Also if these are not for documentation then make them `//` rather than `///`.